changelog.txt

```diff
@@ -1,5 +1,7 @@
 V.Next
 ---------
+- [MINOR] Rename "is QR + PIN available" API to "get preferred auth method". (#2288)
+- [PATCH] Handle Receiver Callback Exception in CommandDispatcher interactive command (#2315)
```

Comment:
Is this change also part of the PR? (2315)?

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2288/commits/4d1c46ae147bf561b8d642aeaa7f9e485931f139